crates/node/src/command.rs

```diff
@@ -218,9 +205,115 @@ pub fn run() -> sc_cli::Result<()> {
             cli.run.run_cmd.network_params.node_key_params.node_key_file =
                 Some((madara_path.clone() + "/p2p-key.ed25519").into());
 
+            if cli.run.genesis_url.is_some() {
+                // can't copy extra genesis-assets atm
+                // we can reuse #982 to create the standard to fetch relevant files
+                let copied = utils::fetch_from_url(
+                    cli.run.genesis_url.clone().unwrap(),
+                    madara_path.clone() + "/configs/genesis-assets",
+                );
+
+                if !copied {
+                    return Err("Failed to fetch genesis assets".into());
+                }
+            } else {
+                // TODO confirm with the CI that we are fetching all and fetch dynamically
+                // Issue #982
+                let files: Vec<&str> =
+                    vec!["Account.json", "AccountBaseImpl.json", "CallAggregator.json", "genesis.json", "Proxy.json"];
+                for file in files {
+                    let src_path: String = utils::get_project_path() + "/configs/genesis-assets/" + file;
+                    let mut copied: bool =
+                        utils::copy_from_filesystem(src_path, madara_path.clone() + "/genesis-assets");
+                    if !copied {
+                        copied = utils::fetch_from_url( "https://raw.githubusercontent.com/keep-starknet-strange/madara/main/configs/genesis-assets/".to_string() + file, madara_path.clone() + "/genesis-assets");
+
+                        if !copied {
+                            return Err("Failed to fetch genesis assets".into());
+                        }
+                    }
+                }
+            }
+
+            // TODO confirm with the CI that we are fetching all and fetch dynamically
+            // Issue #982
+            let files: Vec<&str> = vec![
```

Comment:
This should probably be a constant somewhere

Reply:
any suggestions where to keep all the constants?

Reply:
we can make a constant.rs inside the node crate